witchcraft/internal/tcpjson/envelope.go

```diff
@@ -20,8 +20,8 @@ import (
 )
 
 type LogEnvelopeV1 struct {
-	Metadata LogEnvelopeMetadata `json:",inline"`
-	Payload  json.RawMessage     `json:"payload"`
+	LogEnvelopeMetadata `json:",inline"`
```

Comment:
I think the `json":,inline"` doesn't actually do anything, so might as well remove

Reply:
I saw the issue about that actually which is what made me embed the type, just forgot to remove the tag. thanks!